test_runner/regress/test_pg_regress.py

```diff
@@ -411,6 +411,8 @@ def rollback_and_wait():
     create()
     truncate()
 
+    env.pageserver.allowed_errors.extend(".*removing non-existing aux file.*")
+
```

Comment:
If statistics limit is exceeded, then previous AUX file is removed to avoid it restore on restart.
But this file may be absent. There is no efficient and convenient way to check if key exists during walingest.
So I decided just to accept warning.